editors/code/vscode.proposed.d.ts

```diff
@@ -0,0 +1,1737 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
What is the rationale behind copying it here?

Reply:
Followed instructions for proposed extensions. It will be removed before merging.